### PR TITLE
Fix Duck Array `RuntimeWarning` for EK60 `ep.open_raw`

### PR DESCRIPTION
**Pull request by ctuguinay**
Addresses #1335. I have a step-by-step explanation of the problem and my proposed solution on the linked issue.

### CODE REVIEW

**Comment by ctuguinay** [comment]
@leewujung This is ready for review. Small fix for a warning I kept encountering when trying to do the OOI notebook PR

**Comment by ctuguinay** [comment]
@leewujung I made the change that you proposed in #1335. The failing tests are unrelated to this PR and are the processing level tests mentioned in #1342.

I am curious though, why in

```
EXPECTED_VAR_DTYPE = {
    "channel": np.str_,
    "cal_channel_id": np.str_,
    "beam": np.str_,
    "channel_mode": np.float16,
    "beam_stabilisation": np.byte,
    "non_quantitative_processing": np.int16,
}  # channel name  # beam name
```

are we only checking for these subset of variable types? Are they special in some way?

File: echopype/tests/convert/test_convert_ek60.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
+import warnings
+
 import numpy as np
 import pandas as pd
 from scipy.io import loadmat
-from echopype import open_raw
 import pytest
 
+from echopype import open_raw
+import echopype as ep
```
Comment (leewujung) [inline echopype/tests/convert/test_convert_ek60.py:9]: Seems like the new test uses `open_raw` only, so you don't need this?

**Review by leewujung — DISMISSED** [review]
LGTM! Only a comment on an import statement.

**Comment by leewujung** [comment]
> are we only checking for these subset of variable types? Are they special in some way?

In fact I don't remember exactly why. But I think it was due to similar warnings, since before these types were either not set or set to something that would cause a warning. 

I think we should at some point implement a pydantic approach for all the types, to avoid the messiness and the arbitrary patching approach you pointed out here.

Also note that our NaN-padding approach probably made many of the resultant types different from those written in the convention. Something we should also discuss, probably with the user community and convention folks.

File: echopype/tests/convert/test_convert_ek60.py (hunk 1 above)
Comment (leewujung) [reply echopype/tests/convert/test_convert_ek60.py:9]:
```suggestion
```

**Review by leewujung — APPROVED** [review]

**Pull request merged**